docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# Example file for a Drupal-based setup.
+version: '3'
+services:
+
+  web:
+    image: reload/drupal-apache-fpm
+    ports:
+      - '80'
+    depends_on:
+      - fpm
+    volumes:
+      - './:/var/www/web'
+
+  fpm:
```

Comment:
This is purely form but if we are to execute Drush and other PHP based CLI tools from within this image then I would like to see it named `php`. Running `fpm` it just one of its' use cases.

Reply:
Agreed - fixed!